assets/app.js

```diff
@@ -5,11 +5,39 @@ require( 'oojs-ui/dist/themes/wikimediaui/images/icons/language.svg' );
 require( 'oojs-ui/dist/themes/wikimediaui/images/icons/logo-Wikimedia-Commons.svg' );
 require( 'oojs-ui/dist/themes/wikimediaui/images/icons/download.svg' );
 
-// Hide tutorial box (and keep it hidden).
-$( '.close' ).on( 'click', function () {
-	$( '.tutorial' ).slideUp();
-	window.localStorage.setItem( 'tutorial', 'no' );
+// Set up App namespace.
+global.App = {};
+
+// Load i18n message files.
+$().ready( function () {
```

Comment:
I think this could be written as `$(function () {`, not that it matters much.

Reply:
Good point. Done.